CHANGELOG.md

```diff
@@ -52,38 +52,60 @@
+  The configuration option `instrumentation.thread.tracing` is now enabled by default. This configuration allows the agent to properly monitor code occurring inside threads. In Ruby agent 9.0, instrumented code within threads will be recorded and associated with the current transaction when the thread was created.
 
-- **Add Fiber Instrumentation**
+  This may be a breaking change if you are currently using custom thread instrumentation. New transactions inside of threads will no longer be started if one already exists. [PR#1767](https://github.com/newrelic/newrelic-ruby-agent/pull/1767)
+
+- **Add Fiber instrumentation**
 
   `Fiber` instances are now automatically instrumented similarly to `Thread` instances. This can be configured using `instrumentation.fiber`. [PR#1802](https://github.com/newrelic/newrelic-ruby-agent/pull/1802)
 
 
-- **Ruby 2.2 and 2.3 Dropped**
+- **Removed support for Ruby 2.2 and 2.3**
 
-  Support for Ruby 2.2 and 2.3 dropped with this release. They are no longer included in our test matrices and are not supported
-  for 9.0.0 and onward. [PR#1778](https://github.com/newrelic/newrelic-ruby-agent/pull/1778)
+  Ruby 2.2 and 2.3 are no longer supported by the Ruby agent. To continue using the latest Ruby Agent version, please update to Ruby 2.4.0 or above. [PR#1778](https://github.com/newrelic/newrelic-ruby-agent/pull/1778)
 
-- **Instrumentation dropped for select gems**
+- **Removed deprecated instrumentation**
 
-  Support for the following gems had been dropped. [PR#1788](https://github.com/newrelic/newrelic-ruby-agent/pull/1788)
+  Instrumentation for the following gems had been previously deprecated and has now been removed. [PR#1788](https://github.com/newrelic/newrelic-ruby-agent/pull/1788)
     - Acts As Solr
     - Authlogic
     - DataMapper
     - Rainbows
     - Sunspot
 
+  Versions of the following technologies had been previously deprecated and are no longer supported.
+
+    - Passenger: 2.2.x - 4.0.x
+    - Puma: 2.0.x
+    - Grape: 0.2.0
+    - Padrino: 0.14.x
+    - Rails: 3.2.x
+    - Sinatra: 1.4.x, 1.5.x
+    - Mongo: 1.8.x - 2.3.x
+    - Sequel: 3.37.x, 4.0.x
+    - Delayed_Job: 2.0.x - 4.0.x
+    - Sidekiq: 4.2.x
+    - Excon: below 0.55.0
+    - HttpClient: 2.2.0 - 2.8.0
+    - HttpRb: 0.9.9 - 2.2.1
+    - Typhoeus: 0.5.3 - 1.2.x
+    - Bunny: 2.0.x - 2.6.x 
+    - ActiveMerchant: 1.25.0 - 1.64.x 
+
+
+- **Updated API method `set_transaction_name`**
 
-- **Changes how the API method `set_transaction_name` works**
+  When the method `NewRelic::Agent.set_transaction_name` is called, it will now always change the name and category of the currently running transaction to what is passed into the method. This is a change from previous agent versions. [PR#1797](https://github.com/newrelic/newrelic-ruby-agent/pull/1797)
```

Comment:
Should the PR link be moved to the end of the line beginning with "previously"?

Reply:
[Moved](https://github.com/newrelic/newrelic-ruby-agent/pull/1827/commits/3588b245f177fbe1c38ba66ffe1ae89f36fd600d) :)